build/services/front-types.d.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ Copyright 2016-2017 Resin.io
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import { Front } from 'front-sdk';
+import { ServiceAPIHandle, ServiceEmitContext } from './service-types';
+
```

Comment:
Just realised there's no longer a `FrontEmitRequestContext`.

How do we send general API requests to Front? It appears Flowdock's `FlowdockEmitRequestContext` still exists, and Discourse doesn't have one. NotifyBot (which I assumed you based the initial Front interface from) makes calls using the same pattern as VersionBot does to Github and Flowdock.

At the moment, it appears now the only things you can do with the FrontServiceEmitter is to deal with Comments or Conversations.